dnf-behave-tests/features/steps/cmd.py

```diff
@@ -59,6 +60,19 @@ def faketime_today(context, when):
     context.execute_steps('when I move the clock backward to "{}"'.format(when))
 
 
+@behave.step("I fake kernel release to {release}")
+def i_fake_kernel_release(context, release):
+    libfakeuname_path = '{}/libfakeuname.so'.format(rpm.expandMacro("%{_libdir}"))
+    assert os.path.exists(libfakeuname_path), 'libfakeuname.so library must be installed'
+    context.fake_kernel_release = "FAKE_KERNEL_RELEASE={0} LD_PRELOAD={1} ".format(
```

Comment:
Why not call the `/usr/bin/fakeuname` script I've added to the package just for the reason to make this step easier (and not having to look for the libfakeuname.so file location)?

Reply:
This is so much easier! I didn't notice the script, thanks.

Reply:
Yeah :slightly_smiling_face: I've told you about the script before :slightly_smiling_face: but no worries.

You've taken out the assert completely, maybe we want to keep a simple assert in, the same way `libfaketime` has it above this step? Come think about it, since this is a bit of our speciality, the message should maybe also say the package is in our nightly copr?

Reply:
I added the assert.